tutorials/combine-error-mitigation-options-with-the-estimator-primitive/combine-error-mitigation-options-with-the-estimator-primitive.ipynb

```diff
@@ -33,13 +33,13 @@
    "source": [
     "## Qiskit Pattern workflow\n",
     "\n",
-    "We'll develop our workflow using the four steps of a Qiskit Pattern.\n",
+    "The four steps of a Qiskit Pattern provide a framework for the workflow in this tutorial.\n",
     "\n",
     "### Step 1: Map classical inputs to a quantum problem\n",
     "\n",
-    "For the purposes of this tutorial, we'll assume that our classical problem has already been mapped to quantum, and we'll begin by constructing a circuit and observable to measure. While the techniques used in this tutorial apply to many different kinds of circuits, for simplicity we'll use the [`EfficientSU2`](https://docs.quantum.ibm.com/api/qiskit/qiskit.circuit.library.EfficientSU2#efficientsu2) circuit included in Qiskit's circuit library.\n",
+    "For the purposes of this tutorial, assume that our classical problem has already been mapped to quantum. Begin by constructing a circuit and observable to measure. While the techniques used in this tutorial apply to many different kinds of circuits, for simplicity this tutorial uses the [`EfficientSU2`](https://docs.quantum.ibm.com/api/qiskit/qiskit.circuit.library.EfficientSU2#efficientsu2) circuit included in Qiskit's circuit library.\n",
     "\n",
-    "`EfficientSU2` is a parameterized quantum circuit designed to be efficiently executable on quantum hardware with limited qubit connectivity, while still being expressive enough to solve problems in application domains like optimization and chemistry. It's built by alternating layers of parameterized single-qubit gates with a layer containing a fixed pattern of two-qubit gates, for a chosen number of repetitions. The pattern of two-qubit gates can be specified by the user. We'll use the built-in `pairwise` pattern because it minimizes the circuit depth by packing the two qubit gates as densely as possible. This pattern can be executed using only linear qubit connectivity."
+    "`EfficientSU2` is a parameterized quantum circuit designed to be efficiently executable on quantum hardware with limited qubit connectivity, while still being expressive enough to solve problems in application domains like optimization and chemistry. It's built by alternating layers of parameterized single-qubit gates with a layer containing a fixed pattern of two-qubit gates, for a chosen number of repetitions. The pattern of two-qubit gates can be specified by the user. Herre you can use the built-in `pairwise` pattern because it minimizes the circuit depth by packing the two-qubit gates as densely as possible. This pattern can be executed using only linear qubit connectivity."
```

Comment:
```suggestion
    "`EfficientSU2` is a parameterized quantum circuit designed to be efficiently executable on quantum hardware with limited qubit connectivity, while still being expressive enough to solve problems in application domains like optimization and chemistry. It's built by alternating layers of parameterized single-qubit gates with a layer containing a fixed pattern of two-qubit gates, for a chosen number of repetitions. The pattern of two-qubit gates can be specified by the user. Here you can use the built-in `pairwise` pattern because it minimizes the circuit depth by packing the two-qubit gates as densely as possible. This pattern can be executed using only linear qubit connectivity."
```